ui/lib/replication/addon/components/known-secondaries-card.js

```diff
@@ -0,0 +1,19 @@
+import Component from '@ember/component';
+
+/**
+ * @module KnownSecondariesCard
+ * KnownSecondariesCard components are used on the Replication Details dashboards to display a table of known secondary clusters.
+ *
+ * @example
+ * ```js
+ * <KnownSecondariesCard @cluster={{clusterModel}} @replicationAttrs={{replicationAttrs}} />
+ * ```
+ * @param {object} cluster=null - The cluster model.
+ * @param {string} replicationAttrs=null - The attributes passed directly from the cluster model. These are passed down to the KnownSecondariesTable.
```

Comment:
Below it looks like this is actually an object with a `knownSecondaries` node, not a string

Reply:
ah yes, that's correct. thanks!